library/src/enabled/java/com/telefonica/androidlogger/ui/adapter/LogListAdapter.kt

```diff
@@ -4,41 +4,41 @@ import android.content.ClipData
 import android.content.ClipboardManager
 import android.content.Context
 import android.graphics.Color
+import android.graphics.drawable.GradientDrawable
 import android.view.LayoutInflater
 import android.view.ViewGroup
 import android.widget.Toast
-
-import com.telefonica.androidlogger.R
-
-import androidx.recyclerview.widget.RecyclerView
 import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.RecyclerView
+import com.telefonica.androidlogger.R
 import com.telefonica.androidlogger.ui.holder.LogListItemHolder
 import com.telefonica.androidlogger.ui.viewmodel.AppLoggerViewModel
+import com.telefonica.androidlogger.ui.viewmodel.LogCategoryViewModel
 import com.telefonica.androidlogger.ui.viewmodel.LogEntryViewModel
 import kotlin.math.min
 
 internal class LogListAdapter(
-    private val viewModel: AppLoggerViewModel
+        private val viewModel: AppLoggerViewModel
```

Comment:
I don't about this formatting why it's adding extra space. I'm using the IDE default codestyle